apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -22,16 +23,29 @@ const styles = {
     flexGrow: 1,
     padding: 10,
     borderLeft: 'solid 1px black'
+  },
+  header: {
+    margin: '10px 0px'
+  },
+  navLink: {
+    fontSize: 18,
+    color: color.purple
   }
 };
 
 class LessonOverview extends Component {
   static propTypes = {
-    displayName: PropTypes.string.isRequired,
-    overview: PropTypes.string,
+    lesson: PropTypes.shape({
+      unit: PropTypes.shape({
+        displayName: PropTypes.string.isRequired,
+        link: PropTypes.string.isRequired
+      }),
+      displayName: PropTypes.string.isRequired,
+      overview: PropTypes.string,
+      purpose: PropTypes.string,
+      preparation: PropTypes.string
+    }),
```

Comment:
please make lesson and unit required, as they are accessed unconditionally in the render method.